service/README.md

```diff
@@ -0,0 +1,157 @@
+# nydus-service
+
+The `nydus-service` crate helps to reuse the core services of nydus, allowing you to integrate nydus services into your project elegantly and easily. It provides:
+
+* fuse service
+* virtio-fs service
+* fscache service
+* blobcache service
+
+It also supplies the nydus daemon and the daemon controller to help manage these services.
+
+## Why you need
+
+You're supposed to know that `nydusd` running as daemon to expose a [FUSE](https://www.kernel.org/doc/html/latest/filesystems/fuse.html) mountpoint, a [Virtio-FS](https://virtio-fs.gitlab.io/) mountpoint or an [EROFS](https://docs.kernel.org/filesystems/erofs.html) mountpoint inside guest for containers to access, and it provides key features include:
+
+- Container images are downloaded on demand
+- Chunk level data deduplication
+- Flatten image metadata and data to remove all intermediate layers
+- Only usable image data is saved when building a container image
+- Only usable image data is downloaded when running a container
+- End-to-end image data integrity
+- Compatible with the OCI artifacts spec and distribution spec
+- Integrated with existing CNCF project Dragonfly to support image distribution in large clusters
+- Different container image storage backends are supported
+
+If you want to use these features as native in your project without preparing and invoking `nydusd` deliberately, `nydus-service` is just born for this.
+
+## How to use
+
+For example, reuse the fuse service with `nydus-service` in three steps.
+
+**prepare the config**:
+
+```rust
+{
+  "device": {
+    "backend": {
+      "type": "registry",
+      "config": {
+        "scheme": "",
+        "skip_verify": true,
+        "timeout": 5,
+        "connect_timeout": 5,
+        "retry_limit": 4,
+        "auth": "YOUR_LOGIN_AUTH="
+      }
+    },
+    "cache": {
+      "type": "blobcache",
+      "config": {
+        "work_dir": "cache"
+      }
+    }
+  },
+  "mode": "direct",
+  "digest_validate": false,
+  "iostats_files": false,
+  "enable_xattr": true,
+  "fs_prefetch": {
+    "enable": true,
+    "threads_count": 4
+  }
+}
```

Comment:
Could you please help to update config_v2 format?

Reply:
https://github.com/dragonflyoss/image-service/blob/master/docs/samples/configuration_v2.toml

Reply:
Yeah, I'd like to.

Reply:
@taoohong Hi, have you solved this problem?